velox/common/memory/SharedArbitrator.cpp

```diff
@@ -618,8 +685,22 @@ bool SharedArbitrator::arbitrateMemory(ArbitrationOperation* op) {
   VELOX_CHECK_LT(freedBytes, maxGrowTarget);
 
   RECORD_METRIC_VALUE(kMetricArbitratorSlowGlobalArbitrationCount);
+  // Reclaim by spilling is slow, so we spill to meet only the minimum
```

Comment:
I think we could record the reclaimedBytes in ArbitrationOperation and quit inside reclaimUsedMemoryFromCandidatesBySpill() if meets requestBytes.